pkgs/games/supertux/default.nix

```diff
@@ -1,7 +1,10 @@
-{ lib, stdenv, fetchurl, cmake, pkg-config, SDL2, SDL2_image , curl
-, libogg, libvorbis, libGLU, libGL, openal, boost, glew
-, libpng, freetype, glm
-}:
+{ lib, stdenv, fetchurl, cmake, pkg-config
+, boost, curl
+, SDL2, SDL2_image
+, libSM, libXext
+, libpng, freetype
+, libGLU, libGL, glew, glm
+, openal, libogg, libvorbis }:
```

Comment:
```suggestion
{ lib
, stdenv
, fetchurl
, cmake
, pkg-config
, boost
, curl
, SDL2
, SDL2_image
, libSM
, libXext
, libpng
, freetype
, libGLU
, libGL
, glew
, glm
, openal
, libogg
, libvorbis
}:
```

Reply:
For a large number argument, I find grouping by categories much more readable than spreading it almost a-screen tall.

Reply:
We don't sort them into categories like this especially it is hard to agree what goes into what group and so. Usually we either sort everything alphabetical and put lib and stdenv in the beginning, sort them by *Inputs and there alphabetical or it is not sorted at all. What I suggested also works with the usual formatters, so it will be kept by the next person.

Reply:
Fair enough.